.gitignore/src/main/java/com/task/service/impl/TextServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.task.service.impl;
+
+import com.task.service.TextService;
+import org.springframework.stereotype.Service;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+public class TextServiceImpl implements TextService {
+
+    @Override
+    public List<String> getMostUsedWords(int limit, List<String> sentences) {
+        List<String> mostUsedWords = new ArrayList<>();
+        List<String> words = updateSentences(sentences);
+        Map<String, Long> map = words.stream()
+                .collect(Collectors.groupingBy(word -> word, Collectors.counting()));
+        map.entrySet().stream()

```

Comment:
```suggestion
        return map.entrySet().stream()

```
і тоді не потрібний список  List<String> mostUsedWords